### DISCUSSION TITLE
Setting Manipulator Speed when using IK sovler

### DISCUSSION THREAD

**Discussion by SonYeongGwang** [isaac-sim/IsaacLab#773] [Q&A] [answered]
Hi, I'm in the tutorial for using motion generators from Isaac Lab document 
(https://isaac-sim.github.io/IsaacLab/source/tutorials/05_controllers/run_diff_ik.html#using-a-task-space-controller)

While, using the code, I wondered how to control the speed of the robot. Because the solver only takes accounts for positions I can not find where to give my parameters for velocity setting.

Thanks!

**Comment by Mayankm96** [comment, marked as answer]
You'd need to implement that into the solver. It doesn't exist.
